app/services/witter.js

```diff
@@ -0,0 +1,7 @@
+const axios = require('axios');
```

Comment:
If you're creating an axios file in `config/axios.js` then you don't need to require axios here.

Reply:
config/axios.js does not export axios